src/coreComponents/mainInterface/ProblemManager.cpp

```diff
@@ -139,6 +139,8 @@ ProblemManager::ProblemManager( conduit::Node & root ):
     setRestartFlags( RestartFlags::WRITE ).
     setDescription( "Whether to disallow using pinned memory allocations for MPI communication buffers." );
 
+  chai::ArrayManager::getInstance()->disableCallbacks();
```

Comment:
Won't this globally disable move logging? I thought this was a command line option?

Reply:
It is...but the command line option follows this. I assume it can be enabled after being disabled? Perhaps that is a bad assumption?